src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -506,7 +506,7 @@ void BaseQNAcceleration::iterationsConverged(
 {
   PRECICE_TRACE();
 
-  if (utils::MasterSlave::isMaster() || (not utils::MasterSlave::isMaster() && not utils::MasterSlave::isSlave()))
+  if (utils::MasterSlave::isMaster() || !utils::MasterSlave::isParallel())
```

Comment:
Not sure if it is wise to change from `not` to `!`. I thought the `not` is our coding convention, although I like the `!` more. If you start changing this here you should probably also change it in other places.

Reply:
I try to change the `not` to `!` whenever I touch some code as it will simplify the [port to MSVC](https://stackoverflow.com/a/24414224) in the long run.